addons/purchase_stock/models/purchase.py

```diff
@@ -37,15 +39,15 @@ def _default_picking_type(self):
                  'order_line.move_ids.state',
                  'order_line.move_ids.picking_id')
     def _compute_picking(self):
+        orders = defaultdict(lambda: set())
+        move_ids = self.env['stock.move'].search([('purchase_line_id.order_id', 'in', self.ids)]).ids
```

Comment:
== `self.mapped("order_line.move_ids").ids` (it will use the ORM cache - not sure if it is useful in this case). What do you think ?